libs/providers/flagd/src/lib/service/http/service.ts

```diff
@@ -0,0 +1,162 @@
+import { ErrorCode, EvaluationContext, ResolutionDetails } from "@openfeature/nodejs-sdk";
+import axios, { AxiosResponse } from "axios";
+import { IService } from "../IService";
+import {
+  ResolveBooleanResponse,
+  ResolveNumberResponse,
+  ResolveStringResponse,
+  ResolveObjectResponse
+} from '../../../proto/ts/schema/v1/schema'
+
+
+export default class HTTPService implements IService {
+    private url: string
+
+    constructor(host?: string, port?: number) {
+        if (host == undefined) {
+            host = "localhost"
```

Comment:
```suggestion
            host = "http://localhost"
```
Axios complained about a missing protocol when using the default value.